bqschema_suite_test.go

```diff
@@ -1,10 +1,10 @@
-package bqschema_test
+package bqschema
```

Comment:
Why this change?  Changing the test package this way will give the test package the ability to see all private items instead of just the publish facing elements?

It could be rewritten as import( . "github.com/SeanDolphin/bqschema") if you did not want the explicient call to bqschema?